src/Toggler.js

```diff
@@ -44,9 +44,11 @@ class Toggler extends State {
 			return content;
 		}
 
-		content = header.querySelector(this.content);
-		if (content) {
-			return content;
+		if (header.querySelector) {
```

Comment:
Hi @fernandosouza , I guess I first tried passing in a string of the desired class to `toggle`, it would error out at this line. However, I realize that if we are mandating passing in elements, then this verification is unnecessary. I can remove it if you believe we shouldn't have it